content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -194,13 +194,66 @@ func hello(w http.ResponseWriter, req *http.Request) {
 
 {{% /tab %}}
 
-<!-- {{% tab "NodeJS" %}}
+{{% tab "Node.js" %}}
 
 ## Compatibility requirements
 
+The Datadog Node.js SDK supports library [instrumentations][17] using the OpenTelemetry Node.js Trace API. Follow the detailed setup and example below to instrument a sample application.
+
 ## Setup
 
-{{% /tab %}} -->
+ 1. Follow the instructions in the "Imports and Setup" sections of the [Node.js Custom Instrumentation using OpenTelemetry API][18] page.
+ 2. Complete the steps for instrumenting your service with your selected `opentelemetry-js-contrib` library.
+
+The following example demonstrates how to instrument the `http` and `express` OpenTelemetry integrations with the Datadog Node.js SDK:
+
+```js
+const tracer = require('dd-trace').init()
+const { TracerProvider } = tracer
+const provider = new TracerProvider()
+provider.register()
+
+const { registerInstrumentations } = require('@opentelemetry/instrumentation')
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http')
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express')
+
+// Register the instrumentation with the Datadog trace provider
+// and the OpenTelemetry instrumentation of your choice
+registerInstrumentations({
+  instrumentations: [
+    new HttpInstrumentation({
+      ignoreIncomingRequestHook (req) {
+        // Ignore spans created from requests to the agent
+        return req.path === '/v0.4/traces' || req.path === '/v0.7/config' ||
+        req.path === '/telemetry/proxy/api/v2/apmtelemetry'
+      },
+      ignoreOutgoingRequestHook (req) {
+        // Ignore spans created from requests to the agent
+        return req.path === '/v0.4/traces' || req.path === '/v0.7/config' ||
+        req.path === '/telemetry/proxy/api/v2/apmtelemetry'
+      }
+    }),
+    new ExpressInstrumentation()
+  ],
+  tracerProvider: provider
+})
+
+const express = require('express')
+const http = require('http')
+
+// app code below ....
+```
+
+## Configuration
+
+While not mandatory, it is recommended to disable the corresponding Datadog instrumentation to avoid duplicate spans.
+
+To do this, launch your application with the DD_TRACE_DISABLED_INSTRUMENTATIONS environment variable set to a comma-separated list of integration names to disable. For this application, you would use: DD_TRACE_DISABLED_INSTRUMENTATIONS=http,dns,express,net.
```

Comment:
`````suggestion
To avoid duplicate spans, disable the corresponding Datadog instrumentations.

Set the `DD_TRACE_DISABLED_INSTRUMENTATIONS` environment variable to a comma-separated list of integration names to disable. For example, to disable Datadog instrumentations for the libraries used in the Setup example, set the following:

```sh
DD_TRACE_DISABLED_INSTRUMENTATIONS=http,dns,express,net
```
`````